### PR TITLE
Convert to use ansible-collection-openstack 

### PR DESCRIPTION
**Pull request by scrungus**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ packages. For example:
 
    $ virtualenv venv
    $ source venv/bin/activate
-   $ pip install -U pip
+   $ python -m pip install --upgrade pip
```
Comment (markgoddard) [inline README.rst:20]: Wondering why you needed to change this

**Review by markgoddard — COMMENTED** [review]
The code change looks good. Possibly needs more testing of the collection.

**Comment by markgoddard** [comment]
I tested this (with care) against a production OpenStack deployment. A few things I noticed:

- There is a [bug](https://bugs.launchpad.net/ansible-collections-openstack/+bug/2049658) in the openstack.cloud.router module that prevents it from using an external network by name in a different project. This is probably a blocker, since it's quite a common scenario.
- The Injected file content bytes quota was previously ignored, but now seems to be applied.

File: README.rst (hunk 1 above)
Comment (scrungus) [reply README.rst:20]: was a while ago so i can't really remember. I think it was possibly an issue on my system where `pip` pointed to system level pip, so we use `python -m pip` to use specifically venvs `pip`. FWIW they're functionally equivalent in this case.

**Comment by sd109** [comment]
Proposed workaround for blocking router module bug https://github.com/stackhpc/ansible-collection-openstack/pull/27

**Comment by markgoddard** [comment]
> @markgoddard there's a fix proposed now https://review.opendev.org/c/openstack/ansible-collections-openstack/+/905979, maybe someone on our side can add to the discussion?

I know, I proposed it :laughing:

**Review by markgoddard — APPROVED** [review]

**Pull request merged**